tutteli-gradle-publish/src/main/groovy/ch/tutteli/gradle/publish/PublishPluginExtension.groovy

```diff
@@ -78,7 +78,6 @@ class PublishPluginExtension {
         bintrayPkg = project.objects.property(String)
         bintrayOrganisation = project.objects.property(String)
         signWithGpg = project.objects.property(Boolean)
-        signWithGpg.set(true)
```

Comment:
is it still true per default this way?

Reply:
If the user sets it manually, it will have that value. Otherwise, it will be set to `true` off the project version does not end with `-SNAPSHOT`.

Reply:
OK, learned something new. I was expecting that boolean properties are also `false` per default in gradle.